src/the-2021-scheduler.md

```diff
@@ -24,32 +24,30 @@ Client support:
 
 Because the v3 scheduler uses a different approach to gathering and sorting
 cards, a v2 and v3 client may show a different number of due cards on a given
-day, and may show them in a different order. Please bear this in mind if you're
-using a client that is still only supports v2.
+day, and may show them in a different order.
 
 ## Changes
 
 ### Undo
 
-The v3 scheduler uses Anki's new undo infrastructure - you can answer a card,
-bury a different card, and answer another card, then undo each one in turn if
-you wish. Previous schedulers handled undo separately, so a non-review action
+The v3 scheduler uses Anki's new undo infrastructure: you can answer a card,
+bury another card, answer a different card, then undo each action in turn. Previous schedulers handled undo for review and non-review actions separately, so a non-review action
 would clear the review history, and vice versa.
 
 ### Daily limits
 
-The new count is now capped by the review count by default, so that new card introduction
+The new card limit in v3 scheduler is capped by the review limit by default, so that new card introduction
```

Comment:
```suggestion
By default, the new card limit in the v3 scheduler is capped by the review limit, so that new card introduction
```

Reply:
Is this any better? I don't want to put too much attention here so maybe we can just copy the tooltips:

```suggestion
By default, the review limit in v3 scheduler also applies to new cards, so that new card introduction
```

Reply:
```suggestion
By default, the review limit in the v3 scheduler also applies to new cards, so that new card introduction
```